README.md

```diff
@@ -41,6 +36,13 @@ https://terraform-ibm-modules.github.io/documentation/#/implementation-guideline
 
 ### Usage
 
+  source            = "terraform-ibm-modules/scc/ibm"
+  version           = "X.X.X" # Replace "X.X.X" with a release version to lock into a specific release
+  instance_name     = "my-scc-instance"
+  plan              = "security-compliance-center-standard-plan"
+  region            = "us-south"
+  resource_group_id = "xxXXxxXXxXxXXXXxxXxxxXXXXxXXXXX"
```

Comment:
you missing the backticks around the code block, and so its not rendering in the markdown:
![image](https://github.com/terraform-ibm-modules/terraform-ibm-scc/assets/29608224/1641649b-56b3-4db2-bf34-b21a9d47a145)